src/EA.Weee.Core/AatfReturn/AatfContactData.cs

```diff
@@ -48,7 +48,7 @@ public AatfContactData(Guid id, string firstName, string lastName, string positi
         [Required]
         [StringLength(CommonMaxFieldLengths.Telephone)]
         [Display(Name = "Phone")]
-        [GenericPhoneNumber(ErrorMessage = "The telephone number can use numbers, spaces and some special characters (-+). It must be no longer than 20 characters.")]
```

Comment:
@stevelee-civica  Dont worry about changing this one. Its a legacy screen so dont need to change.

Reply:
rolled back